Assignment-1/code/index.html

```diff
@@ -15,6 +15,16 @@
     <script type="text/babel">
 //React Code here
  ReactDOM.render(<div></div>,document.getElementById('root'));
+ const Alldivs=
+ React.createElement('div',{},
+ React.createElement('div',{class:"div1"}, '1st div'),
+ React.createElement('div',{class:"div2"},'2nd div'),
+ React.createElement('div',{class:"div3"},'3rd div'),
+ React.createElement('div',{class:"div4"},'4th div'),
+ React.createElement('div',{class:"div5"},'5th div'),
+ React.createElement('div',{class:"div6"},'6th div'),
+ );
+ ReactDOM.render(Alldivs,document.getElementById('root'));
```

Comment:
You are using ReactDOM.render property 2 times which I think is not the correct way.